### ISSUE TITLE
Suggestion: Show version in web interface

### ISSUE THREAD

**Issue by ThomasLandauer** [sj26/mailcatcher#479]
This is the only (minor) left-over of https://github.com/sj26/mailcatcher/issues/436

I'm suggesting to display the version in the web interface after the name (top left corner): "MailCatcher 0.8.1" instead of "MailCatcher".

But since the version number is now shown in various other places, it's not that important anymore ;-)

**Comment by sj26** [comment]
Yeah I was leaving #436 open, thinking to add an “about” dialog when “mailcatcher” is clicked which shows the version. But we can track that here.

Thanks!
